tests/lib/config/config-initializer.js

```diff
@@ -183,19 +183,19 @@ describe("configInitializer", function() {
             it("should support the google style guide", function() {
                 var config = init.getConfigForStyleGuide("google");
 
-                assert.deepEqual(config, {extends: "google"});
+                assert.deepEqual(config, {extends: "google", "installedESLint": true });
             });
 
             it("should support the airbnb style guide", function() {
                 var config = init.getConfigForStyleGuide("airbnb");
 
-                assert.deepEqual(config, {extends: "airbnb", plugins: ["react"]});
+                assert.deepEqual(config, {extends: "airbnb", "installedESLint": true, plugins: ["react"]});
             });
 
             it("should support the standard style guide", function() {
                 var config = init.getConfigForStyleGuide("standard");
 
-                assert.deepEqual(config, {extends: "standard", plugins: ["standard"]});
+                assert.deepEqual(config, {extends: "standard", "installedESLint": true, plugins: ["standard"]});
```

Comment:
Thanks for catching that - should definitely be consistent with what's already there.